shared/login/relogin/container.tsx

```diff
@@ -4,19 +4,20 @@ import * as ProvisionGen from '../../actions/provision-gen'
 import * as SignupGen from '../../actions/signup-gen'
 import HiddenString from '../../util/hidden-string'
 import Login from '.'
-import {connect, isNetworkErr} from '../../util/container'
+import {connect, TypedState, TypedDispatch, isNetworkErr} from '../../util/container'
+import * as ConfigTypes from '../../constants/types/config'
```

Comment:
prefer `Types`

Reply:
`Types` is only when it's the same component.